theme/snippets/add-to-cart-container.liquid

```diff
@@ -2,32 +2,45 @@
 {% capture button_label %}{{"snippets.add_to_cart_container.add_to_cart_btn.text" | t }}{% endcapture %}
 {% assign button_disabled = false %}
 {% if product.available %}
-  {% if product_in_cart %}
-    {% capture button_label %}{{"snippets.add_to_cart_container.added_to_cart_btn.text" | t }}{% endcapture %}
-    {% assign button_disabled = true %}
-  {% endif %}
-{% else %}
   {% case product_type %}
     {% when "Course" %}
     {% when "Event" %}
       {% capture button_label %}{{"snippets.add_to_cart_container.registration_closed_btn.text" | t }}{% endcapture %}
       {% assign button_disabled = true %}
+  {% endcase %}
     {% else %}
       {% capture button_label %}{{"snippets.add_to_cart_container.sold_out_btn.text" | t }}{% endcapture %}
       {% assign button_disabled = true %}
-  {% endcase %}
 {% endif %}
 
 <div class="AddToCartContainer">
   <form data-add-to-cart-form data-product-type={{product_type}} action="/cart/add" method="post" enctype="multipart/form-data" id="AddToCartForm">
-    <select class="{% if product.variants.size <= 1 %} none {% endif %}" name="variant" data-variant-selector>
-    {% for variant in product.variants %}
-      <option value="{{ variant.id }}">
-        {{ variant.title }}
-      </option>
-    {% endfor %}
-  </select>
-    <button data-variant-id={{product.first_available_variant.id}} data-add-to-cart-button {% if button_disabled %} disabled {% endif %} class="w100 color-white uppercase sans-light-sm radius-xs bg-color-black py_75" type="submit" name="add" id="AddToCart">
+    <div class="AddToCartContainer__quantity-select-wrapper overflow-hidden relative {% if product.variants.size <= 1 %} mb1_75 {% endif %}">
+      <label for="quantity" type="number" name="quantity" aria-label={{"snippets.add_to_cart_container.quantity_spinner.label" | t }}></label>
+      <select class="sans-xs uppercase radius-xs border-black w100 pl1_125 py_5" name="quantity" id="Quantity" data-quantity-selector>
+        <option value="" selected="selected">{{"snippets.add_to_cart_container.quantity_spinner.label" | t }}</option>
+        {% comment %}
+        TO-DO: Add notification if user adds quantity > product inventory
+        {% endcomment %}
```

Comment:
@sepowitz and I talked about adding this to-do in the case that the added quantity is > product inventory, so the user is notified that they can't add X quantity.

Reply:
Are you able to use liquid to get the available inventory and just disable Quantity options that exceed that amount?